src/ht.jl

```diff
@@ -30,14 +30,11 @@ function ht_svymean(x::Symbol, design)
     total_df = ht_svytotal(x, design)
     total = total_df.total[1]
     var_total = total_df.se[1]
-    
     mean = 1.0 ./ sum(design.popsize) .* total
+    # TODO check standard error, it is incorrect?
```

Comment:
Is it incorrect or not?

Reply:
As i wrote in the issue for ht.jl, `ht_svymean` I was unable to find the correct scaling factor yet... So its not working right now